src/components/sponsors/styles.module.scss

```diff
@@ -0,0 +1,170 @@
+.sponsors {
+    .background{
+        background-image: linear-gradient(to bottom, #196799, #27A4B6);
+        position: relative;
+        min-height: 200vh;
+        z-index: 2;
+        margin-bottom: 0;
+        @media only screen and (max-width: 768px) {
+             min-height: 100vh;
+          }
+          @media only screen and (min-width: 1000px) {
+            min-height: 250vh;
+         }
+         
+          
+
+        .rect{
+            position: absolute;
+            width: 57%;
+            height: auto;
+            left: 25%;
+            top: 14%; //21%
+            text-align: center;
+            border-radius: 40px;
+            transform: rotate(2.49deg);
+    
+    
+    background: #1D2644;
+    border-radius: 20px;
+    transform: matrix(1, -0.05, 0.02, 1, 0, 0);
+    
+    .h1{
+      
+        align-items: center;
+        justify-content: center;
+        font-family: 'Montserrat';
+        font-style: normal;
+        text-align: center;
+        margin: 26px;
+        
+        font-weight: 600;
+        font-size: 70px;
+        line-height: 85px;
+        transform: matrix(1, -0.05, 0.02, 1, 0, 0);
+        color: #C9D2E5;
+    }
+    }
+    
+    .emailBox {
+        position: absolute;
+        background-color:#0D387C8F;
+        width: auto;
+        height: auto;
+        border: 1px solid #0D387C8F ;
+        border-radius: 40px;
+        top: 18%; //60%
+        text-align: center; 
+    
+    }
+    .sponsorstitle {
+        text-align: center;
+        text-transform: uppercase; /* convert the text to uppercase */
+        font-size: 45px;
+        font-weight: bold;
+        margin-top: 1px; /* add some margin to separate the title from the rectangle */
+        
+    }
+    
+    .email{
+        font-family: 'Montserrat';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 50px;
+        line-height: 61px;
+        position: relative;
+        margin: 40px;
+        color: #CAD2E5;
+        top:35% //25%
+        
+    }
+
+}
+.gridContainer{
+    display: grid;
+    grid-template-areas: 
+        "john john"
+        "ghub Epic"
+        "Cardano Cardano"
+        "log Caterpillar"
+        "BLUES BLUES"
+
+
+}
+.logo1{
```

Comment:
The John Deere logo seems a bit pixelated. What about making the company logos smaller so they seem clearer to the website user?